apps/portal/src/components/global/ProductNavigation.tsx

```diff
@@ -58,7 +59,9 @@ export default function ProductNavigation({ items, title, titleHref }: Props) {
                               active ? 'bg-slate-100' : '',
                               'block px-4 py-2 text-sm text-slate-700',
                             )}
-                            href={child.href}>
+                            href={child.href}
+                            rel={item.target ? 'noopener noreferrer' : ''}
```

Comment:
item.target can be `_self` (although we prevent it with TypeScript), which won't require `noopener noreferrer`. It's better to check if the link is an external link and only add the `rel` if so:

```
rel={!child.href.startsWith('/') ? 'noopener noreferrer' : undefined}
```

Using `undefined` is better as it will not render any empty value